modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -91,6 +93,17 @@
 								}).removeClass('firstScreen');
 								_this.hideDisplay();
 							}
+
+							// This plugin uses 'mw.KCuePoints' to retrieve que points, to persist the same conditions
+							// we listen to the same events as 'mw.KCuePoints'
+							$(_this.embedPlayer).bind(
```

Comment:
@esakal if this is the same as kcuepoints why do we need an additional monitor? I would agree that seek operation reshuffles state and also kcuepoint service only updates the last closest cuepoint after a seek, but for all other use cases you can get it from cuepoint event.
If we don't fire code cuepoint event form kcuepoint it's because it wasnt required till now, but again, new requirements new code so we can remedy this in cuepoint service.